WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -691,11 +693,6 @@ title_despot = {
 			has_landed_title = e_byzantium
 			has_landed_title = e_roman_empire
 		}
-		OR = {
-			religion_group = christian
-			religion_group = jewish_group
-			religion_group = pagan_group
-		}
```

Comment:
@loup99 

Those titles were Roman titles. The three true Roman Empires do not cease to be special because of the leader. In the hypothetical scenario that a follower of an unusual religion seizes the Roman Empire, he would most likely continue using Roman titles and traditions to legitimatize his rule. 
Doesn't everyone want to be Emperor of the Romans?

"Keeper of the King's Wardrobe" (etc.) is hardly a religious title. Caesar is a secular title used for the Roman Emperor. The only religious Roman title I can think is "Pontifex", which only Catholics and pagans used.